Examples/Swift/ViewController.swift

```diff
@@ -255,6 +255,8 @@ extension ViewController: WaypointConfirmationViewControllerDelegate {
 
 extension ViewController: NavigationViewControllerDelegate {
     func navigationViewController(_ navigationViewController: NavigationViewController, didArriveAt waypoint: Waypoint) {
+        if waypoint == currentRoute?.legs.last?.destination { showEndOfRouteFeedback(for: waypoint)}
```

Comment:
[If this is set to true](https://github.com/mapbox/mapbox-navigation-ios/blob/feb895f0527c20c06d9a05a539e78ad503086601/Examples/Swift/ViewController.swift#L241), will this automatically be shown?